test/Twilio.Test/Http/SystemNetHttpClientTest.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Castle.Core.Logging;
+using NUnit.Framework;
+using Twilio.Http;
+using HttpMethod = Twilio.Http.HttpMethod;
+
+namespace Twilio.Tests.Http
+{
+    public class MockResponseHandler : DelegatingHandler
```

Comment:
Wrap this whole file with `#if !NET35` (just like `src/Twilio/Http/SystemNetHttpClient.cs` is now) -- .NET 3.5 build is failing on AppVeyor.